test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceOptionsTests.cs

```diff
@@ -72,7 +72,7 @@ INSERT @returntable
         private DeploymentOptions GetIgnoreColumnOptions()
         {
             var options = new DeploymentOptions();
-            options.IgnoreColumnOrder = new DeploymentOptionProperty<bool>(true);
+            options.BooleanOptionsDictionary["IgnoreColumnOrder"].Value = true;
```

Comment:
For all of these hardcoded strings it'd be preferable to use nameof when possible, for example this one should be able to be `nameof(DacDeployOptions.IgnoreColumnOrder)`